tests_integration/testing_utils/snowpark_services_utils.py

```diff
@@ -73,10 +72,11 @@ def logs_should_return_service_logs(self, service_name: str) -> None:
                 "--schema",
                 self.schema,
             ],
-            connection="spcs",
         )
         assert result.output
-        assert result.output.strip() == f"{service_name}/0 Hello World!"
+        # Assert this instead of full payload due to log coloring
```

Comment:
What does log coloring mean?

Reply:
The output of snowcli is generate by rich wich has some auto coloring feature. So you have escape codes in the string